src/api/teams.rs

```diff
@@ -0,0 +1,151 @@
+
+    /// Gets a team from its slug.
+    /// ```no_run
+    /// # async fn run() -> octocrab::Result<()> {
+    /// let team = octocrab::instance()
+    ///     .teams("owner")
+    ///     .get("team")
+    ///     .await?;
+    /// # Ok(())
+    /// # }
+    /// ```
+    pub async fn get(&self, team_slug: impl Into<String>) -> Result<models::Team> {
+        let url = format!(
+            "/orgs/{org}/teams/{team}",
+            org = self.owner,
+            team = team_slug.into(),
+        );
+        self.crab.get(url, None::<&()>).await
+    }
+
+    /// Creates a new team in the organization.
+    /// ```no_run
+    /// # async fn run() -> octocrab::Result<()> {
+    /// use octocrab::params;
+    ///
+    /// octocrab::instance()
+    ///     .teams("owner")
+    ///     .create("new-team")
+    ///     .description("My team created from Octocrab!")
+    ///     .maintainers(&vec![String::from("ferris")])
+    ///     .repo_names(&vec![String::from("crab-stuff")])
+    ///     .privacy(params::teams::Privacy::Closed)
+    ///     .parent_team_id(1u64)
+    ///     .send()
+    ///     .await?;
+    /// # Ok(())
+    /// # }
+    /// ```
+    pub fn create(&self, name: impl Into<String>) -> CreateTeamBuilder {
+        CreateTeamBuilder::new(self, name.into())
+    }
+
+    /// Creates a new team in the organization.
+    /// ```no_run
+    /// # async fn run() -> octocrab::Result<()> {
+    /// use octocrab::params;
+    ///
+    /// octocrab::instance()
+    ///     .teams("owner")
+    ///     .edit("some-team", "Some Team")
+    ///     .description("I edited from Octocrab!")
+    ///     .privacy(params::teams::Privacy::Secret)
+    ///     .parent_team_id(2u64)
+    ///     .send()
+    ///     .await?;
+    /// # Ok(())
+    /// # }
+    /// ```
+    pub fn edit(&self, team_slug: impl Into<String>, name: impl Into<String>) -> EditTeamBuilder {
+        EditTeamBuilder::new(self, team_slug.into(), name.into())
+    }
+
+    /// Deletes a team from the organization.
+    /// ```no_run
+    /// # async fn run() -> octocrab::Result<()> {
+    /// octocrab::instance().teams("owner").delete("some-team").await?;
+    /// # Ok(())
+    /// # }
+    /// ```
+    pub async fn delete(&self, team_slug: impl Into<String>) -> Result<()> {
+        let url = format!(
+            "/orgs/{org}/teams/{team}",
+            org = self.owner,
+            team = team_slug.into(),
+        );
+        self.crab.delete(url, None::<&()>).await
+    }
+
+    /// List the child teams of a team in the organization.
+    /// ```no_run
+    /// # async fn run() -> octocrab::Result<()> {
+    /// # let octocrab = octocrab::Octocrab::default();
+    /// octocrab::instance()
+    ///     .teams("owner")
+    ///     .children("parent-team")
+    ///     .per_page(5)
+    ///     .page(1u8)
+    ///     .send()
+    ///     .await?;
+    /// # Ok(())
+    /// # }
+    /// ```
+    pub fn children(&self, team_slug: impl Into<String>) -> ListChildTeamsBuilder {
```

Comment:
The builder/API methods should include the verb, to distinguish it from handler methods like `repos`.
```suggestion
    ///     .list_children("parent-team")
    ///     .per_page(5)
    ///     .page(1u8)
    ///     .send()
    ///     .await?;
    /// # Ok(())
    /// # }
    /// ```
    pub fn list_children(&self, team_slug: impl Into<String>) -> ListChildTeamsBuilder {
```